versioned_docs/version-latest/faq.md

```diff
@@ -6,14 +6,14 @@ This FAQ is a work in progress designed to answer the questions our users most f
 
 #### **Q: Is Rancher Desktop a desktop version of Rancher?**
 
-**A:** While [Rancher](https://rancher.com/) and [Rancher Desktop](https://rancherdesktop.io/) share the _Rancher_ name, they do different things. Rancher Desktop is not Rancher on the Desktop. Rancher is a powerful solution to manage Kubernetes clusters. Rancher Desktop runs local Kubernetes and a container management platform. The two solutions complement each other.
+**A:** While [Rancher](https://rancher.com/) and [Rancher Desktop](https://rancherdesktop.io/) share the _Rancher_ name, they do different things. Rancher Desktop is not Rancher on the Desktop. Rancher is a powerful solution to manage Kubernetes clusters. Rancher Desktop allows users to locally run a container management platform optionally including Kubernetes. The two solutions complement each other.
```

Comment:
```suggestion
**A:** While [Rancher](https://rancher.com/) and [Rancher Desktop](https://rancherdesktop.io/) share the _Rancher_ name, they do different things. Rancher Desktop is not Rancher on the Desktop. Rancher is a complete software stack for managing multiple Kubernetes clusters across any infrastructure. On the other hand, Rancher Desktop provides Kubernetes and container management on the desktop to support local development use cases. The two solutions complement each other.
```

Reply:
Hmm. The first half is definitely fine.

The second part bothers me. I know that some people use Rancher Desktop for Kubernetes, but most of my team just uses it for Docker, i.e. we've turned off Kubernetes. (The fact that my team is slowly turning it back on is out of scope for this thought.)

I'm also going to reword the thing to start with "No."

```suggestion
**A:** No. Rancher Desktop is not Rancher on the Desktop. 

While [Rancher](https://rancher.com/) and [Rancher Desktop](https://rancherdesktop.io/) share the _Rancher_ name, they do different things. But, the two solutions do complement each other:

Rancher is a complete software stack for managing multiple Kubernetes clusters across any infrastructure.

Rancher Desktop provides container management and an optional Kubernetes instance on the desktop to support local development use cases.
```